.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+# Related Issue
+- the issue goes here
+
+# Summary
+
+# Proposed Changes
+- change 1
+- change 2
+
+# Screenshots (if appropriate)
+
+Original                |         Updated
+:------------------------:|:------------------------:
+** original screenshot ** | ** updated screenshot **|
```

Comment:
This is cool though I'm not sure if images can be added in a way that they are presented side by side on Github. Can you please confirm?

Reply:
![side-by-side](https://user-images.githubusercontent.com/60931421/92300883-3dcc3e00-ef7c-11ea-9fb5-c64e3174a5e3.png)

This is how it looks, one can look closer by clicking on the image individually.

Reply:
Oh! This is so cool, I didn't know about this. Thanks, Sahil!